### PR TITLE
testing digit

### PR DESCRIPTION
**Pull request by atefehrahimi**
rewrite testing for digit

### CODE REVIEW

File: testing.py [hunk 1]
```diff
@@ -1,22 +1,15 @@
-from unittest import TestCase
+import unittest
+from validation_digit import number_validation
 
-import lib
```
Comment (shivai) [inline testing.py]: Why did you delete it?

File: testing.py [hunk 2]
```diff
@@ -1,22 +1,15 @@
-from unittest import TestCase
```
Comment (shivai) [inline testing.py]: Same here

File: testing.py [hunk 3]
```diff
@@ -1,22 +1,15 @@
...
+from validation_digit import number_validation
 
-import lib
 
+class TestDigit(unittest.TestCase):
+    def test_digit_validation(self):
+        sample1 = "123445"                            # a sample1 for check function if return True
+        self.assertTrue(number_validation(sample1))
 
-class TestLib(TestCase):
```
Comment (shivai) [inline testing.py]: Why did you delete this class?

File: testing.py [hunk 4]
```diff
@@ -1,22 +1,15 @@
-from unittest import TestCase
+import unittest
+from validation_digit import number_validation
```
Comment (shivai) [inline testing.py]: The "number_validation" function is in lib.py. What is validation_digit?

File: testing.py [hunk 5]
```diff
@@ -1,22 +1,15 @@
...
-        sample_3 = "me@morteza.123"
-        self.assertFalse(lib.email_validation(sample_3))
-
-        sample_4 = "me@morteza12@.com"
-        self.assertFalse(lib.email_validation(sample_4))
-
-        sample_5 = "me@motezana.com"
-        self.assertTrue(lib.email_validation(sample_5))
 
+if __name__ == '__main__':
```
Comment (shivai) [inline testing.py]: Why did you add this line ?

File: testing.py (hunk 1 above)
Comment (atefehrahimi) [reply testing.py]: I exchange the name to the digit number in my file , I correct it .

File: testing.py (hunk 5 above)
Comment (atefehrahimi) [reply testing.py]: because , when I execute it , the lib.py do not execute.

File: testing.py (hunk 4 above)
Comment (atefehrahimi) [reply testing.py]: Yes , I don't update lib.py.
the name of my file was validation_digit , I should correct it .
